### DISCUSSION TITLE
get error conflict install pponnxcr

### DISCUSSION THREAD

**Discussion by maslota1sa** [cpp-for-yourself/lectures-and-homeworks#77] [Homeworks Q&A] [answered]
it's say conflict
![Screenshot 2024-01-22 205055](https://github.com/cpp-for-yourself/lectures-and-homeworks/assets/157382805/b65ef95b-ef1b-425a-a0ef-2f69b433fdc9)
i checked that onnxruntime not found in my laptop and i can't install it by cmd ( pip install onnxruntime)

**Comment by niosus** [comment]
Sorry, I'm missing context. What do you need pip for in the first place?

**Reply by maslota1sa** [reply]
https://github.com/NextStalker/BBAuto to install BBAuto it's requirement Onnxrunetime and pponnxcr , i just use pip to install Onnxrunetime and pponnxrc 
![image](https://github.com/cpp-for-yourself/lectures-and-homeworks/assets/157382805/fe2a4cd6-f6ec-4dac-9703-2712dc0e302e)

**Reply by niosus** [reply]
Are you sure that you're asking this in the right place? I don't have any relationship with that project 😅

**Reply by niosus** [reply, marked as answer]
I think I'll close this thread but please feel free to reopen if needed.
